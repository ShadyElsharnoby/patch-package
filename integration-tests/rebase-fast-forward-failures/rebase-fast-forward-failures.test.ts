import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "rebase-fast-forward-failures",
  shouldProduceSnapshots: true,
})
