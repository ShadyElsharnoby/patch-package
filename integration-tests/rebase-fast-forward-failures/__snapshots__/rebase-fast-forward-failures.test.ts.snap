// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test rebase-fast-forward-failures: 00: when continuing the rebase, the final patch should fail to apply because it's out of date 1`] = `
"SNAPSHOT: when continuing the rebase, the final patch should fail to apply because it's out of date
patch-package 0.0.0
• Creating temporary folder
• Installing left-pad@1.3.0 with npm
• Diffing your files with clean files
✔ Created file patches/left-pad+1.3.0+002+world.patch

Fast forwarding...

⛔ ERROR

Failed to apply patch file left-pad+1.3.0+003+goodbye.patch.

If this patch file is no longer useful, delete it and run

  patch-package

To partially apply the patch (if possible) and output a log of errors to fix, run

  patch-package --partial

After which you should make any required changes inside node_modules/left-pad, and finally run

  patch-package left-pad

to update the patch file.

END SNAPSHOT"
`;

exports[`Test rebase-fast-forward-failures: 01: when continuing the rebase, the final patch should apply 1`] = `
"SNAPSHOT: when continuing the rebase, the final patch should apply
patch-package 0.0.0
• Creating temporary folder
• Installing left-pad@1.3.0 with npm
• Diffing your files with clean files
✔ Created file patches/left-pad+1.3.0+003+goodbye.patch

END SNAPSHOT"
`;

exports[`Test rebase-fast-forward-failures: 02: the patches should go from 'use strict' to 'use hello' to 'use universe' to 'goodbye universe' 1`] = `
"SNAPSHOT: the patches should go from 'use strict' to 'use hello' to 'use universe' to 'goodbye universe'
diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index e90aec3..1a2ec5f 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -3,7 +3,7 @@
      * and/or modify it under the terms of the Do What The Fuck You Want
      * To Public License, Version 2, as published by Sam Hocevar. See
      * http://www.wtfpl.net/ for more details. */
-'use strict';
+'use hello';
 module.exports = leftPad;
 
 var cache = [
diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index 1a2ec5f..f2e3912 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -3,7 +3,7 @@
      * and/or modify it under the terms of the Do What The Fuck You Want
      * To Public License, Version 2, as published by Sam Hocevar. See
      * http://www.wtfpl.net/ for more details. */
-'use hello';
+'use universe';
 module.exports = leftPad;
 
 var cache = [
diff --git a/node_modules/left-pad/index.js b/node_modules/left-pad/index.js
index f2e3912..a02f494 100644
--- a/node_modules/left-pad/index.js
+++ b/node_modules/left-pad/index.js
@@ -3,7 +3,7 @@
      * and/or modify it under the terms of the Do What The Fuck You Want
      * To Public License, Version 2, as published by Sam Hocevar. See
      * http://www.wtfpl.net/ for more details. */
-'use universe';
+'goodbye universe';
 module.exports = leftPad;
 
 var cache = [
END SNAPSHOT"
`;
